ui/packages/atlasmap/src/UI/dialogs/CustomClassDialog.tsx

```diff
@@ -39,25 +39,31 @@ export interface ICustomClassDialogProps {
 }
 export const CustomClassDialog: FunctionComponent<ICustomClassDialogProps> = ({
   title,
-  value: initialValue = "",
+  customClassName: initialCustomClassName = "",
+  customClassNames,
   collectionType: initialCollectionType = "",
   collectionTypeOptions,
   isOpen,
   onCancel,
   onConfirm,
 }) => {
-  const [value, setValue] = useState(initialValue);
+  const [customClassName, setCustomClassName] = useState(
+    initialCustomClassName,
```

Comment:
Maybe remove comma since you're not providing a callback

Reply:
I'd agree but I think prettier objects... :)